run-dljc.sh

```diff
@@ -6,14 +6,16 @@
 ### Usage
 
 # - Move this script to an experiment directory.
-# - Make a file containing a list of git repositories, one per line. Repositories must be of the form: https://github.com/username/repository - the script is reliant on the number of slashes, so excluding https:// is an error.
+# - Make a file containing a list of git repositories, one per line.
+#   Repositories must be of the form: https://github.com/username/repository
+#   The script is reliant on the number of slashes, so excluding https:// is an error.
 # - Ensure that your JAVA8_HOME variable points to a Java 8 JDK
 # - Ensure that your JAVA11_HOME variable points to a Java 11 JDK
 # - Ensure that your JAVA_HOME variable points to the same place as JAVA11_HOME
 # - Ensure that your CHECKERFRAMEWORK variable points to a built copy of the Checker Framework
 # - Other dependencies: perl, python2.7, awk, git, mvn
 # - Then run a command like the following (replacing the example arguments with your own):
-#   > bash run-dljc.sh -o outdir -i describe-images-list -c org.checkerframework.checker.builder.TypesafeBuilderChecker -l /homes/gws/kelloggm/image-sniping-oss/typesafe-builder-checker/build/libs/typesafe-builder-checker.jar:/homes/gws/kelloggm/.m2/repository/org/springframework/spring-expression/5.1.7.RELEASE/spring-expression-5.1.7.RELEASE.jar:/homes/gws/kelloggm/.m2/repository/org/springframework/spring-core/5.1.7.RELEASE/spring-core-5.1.7.RELEASE.jar:/homes/gws/kelloggm/.m2/repository/org/springframework/spring-jcl/5.1.7.RELEASE/spring-jcl-5.1.7.RELEASE.jar: -s /homes/gws/kelloggm/image-sniping-oss/typesafe-builder-checker/stubs
+#   > bash run-dljc.sh -o outdir -i describe-images-list -c org.checkerframework.checker.builder.TypesafeBuilderChecker -l ${HOME}/image-sniping-oss/typesafe-builder-checker/build/libs/typesafe-builder-checker.jar:${HOME}/.m2/repository/org/springframework/spring-expression/5.1.7.RELEASE/spring-expression-5.1.7.RELEASE.jar:${HOME}/.m2/repository/org/springframework/spring-core/5.1.7.RELEASE/spring-core-5.1.7.RELEASE.jar:${HOME}/.m2/repository/org/springframework/spring-jcl/5.1.7.RELEASE/spring-jcl-5.1.7.RELEASE.jar: -s ${HOME}/image-sniping-oss/typesafe-builder-checker/stubs
```

Comment:
Why does this example command include `-s ${HOME}/image-sniping-oss/typesafe-builder-checker/stubs`?  I would think that the checker implementation would always include this.